src/components/LeftSidebar/LeftSidebar.spec.js

```diff
@@ -612,26 +614,27 @@ describe('LeftSidebar.vue', () => {
 			})
 
 			test('shows group conversation dialog when clicking search result', async () => {
-				const eventHandler = jest.fn()
-				EventBus.$once('new-group-conversation-dialog', eventHandler)
-
 				const wrapper = await testSearch(SEARCH_TERM, [...groupsResults], [])
 
 				const resultsListItems = findNcListItems(wrapper, groupsResults.map(item => item.label))
 				expect(resultsListItems.exists()).toBeTruthy()
 				expect(resultsListItems).toHaveLength(groupsResults.length)
 
 				await resultsListItems.at(1).findAll('a').trigger('click')
-				expect(eventHandler).toHaveBeenCalledWith(groupsResults[1])
+				// Wait for the component to render
+				await wrapper.vm.$nextTick()
+				const ncModalComponent = wrapper.findComponent(NcModal)
+				expect(ncModalComponent.exists()).toBeTruthy()
+
+				const input = ncModalComponent.find('input[placeholder="Enter a name for this conversation"]')
```

Comment:
It doesn't work because it is not rendered as a NcTextField element 
`<div data-v-f50b337c="" class="input-field__main-wrapper"><input data-v-f50b337c="" id="inputxhsjs" type="text" placeholder="Enter a name for this conversation" aria-describedby="" aria-live="polite" trailingbuttonicon="close" class="input-field__input">`

the only "identifier" I could find was obviously the placeholder.